k8s/tf/70_prod_db_redis/main.tf

```diff
@@ -19,13 +19,14 @@ module "mysql-prod" {
   mysql_username              = "root"
   mysql_password              = var.mysql_prod_password
   mysql_identifier            = "sumo"
-  mysql_instance_class        = "db.m5.4xlarge"
+  mysql_instance_class        = "db.m5.2xlarge"
   mysql_backup_retention_days = 14
   mysql_security_group_name   = "sumo_rds_sg_prod"
   mysql_storage_gb            = 250
   mysql_storage_type          = "gp2"
+  mysql_allow_major_version_upgrade = true
```

Comment:
Can you run `terraform fmt -recursive` to fix the spacing?

Reply:
sure!